### ISSUE TITLE
"generate operation" error "the required flag `-n, --name' was not specified"

### ISSUE THREAD

**Issue by borisovg** [go-swagger/go-swagger#2279]
## Problem statement
```
$ ./swagger generate operation
the required flag `-n, --name' was not specified
```
According to `--help` this defaults to all, which suggests that this option should not be required.

## Swagger specification
```
$ cat swagger.yml 
consumes:
- application/json
info:
  title: Swagger
  version: 0.1.0
paths:
  /foo:
    get:
      responses:
        200:
          description: Successful response
          schema:
              type: object
              required:
                - data
              properties:
                data:
                  type: string
produces:
- application/json
schemes:
- http
swagger: "2.0"
```
```
$ ./swagger validate swagger.yml 
2020/04/26 20:22:27 
The swagger spec at "swagger.yml" is valid against swagger specification 2.0
```

## Environment
```
$ ./swagger version
version: v0.23.0
commit: 3596b40bedd4f06cc918767098568456d2ec2370
```
```
$ go version
go version go1.14.2 linux/amd64
```

**Comment by fredbi** [comment]
Yeah fixed that too with #2327. Thanks for reporting. These kinds of minor glitches are always annoying.

**Comment by fredbi** [comment]
fixed by #2327
